tests/foreman/api/test_errata.py

```diff
@@ -55,6 +54,8 @@
 
 CUSTOM_REPO_URL = FAKE_6_YUM_REPO
 CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
+DISTRO_RHEL6 = settings.distro.image_el6
```

Comment:
I tried to add `from robottelo.config import settings` to constants.py as the first option and it didn't work , I got the DISTRO_RHEL6 and DISTRO_RHEL7 values as `None`.